roles/confluent.common/tasks/main.yml

```diff
@@ -16,14 +16,14 @@
     - ansible_distribution
 
 # JVM Installation and Repo Setup
-- include_tasks: redhat.yml
-  when: ansible_os_family == "RedHat"
 
-- include_tasks: ubuntu.yml
-  when: ansible_distribution == "Ubuntu"
+- include_tasks: "{{ ansible_distribution }}.yml"
 
-- include_tasks: debian.yml
-  when: ansible_distribution == "Debian"
+- name: Unset confluent features if community version
```

Comment:
So this ensures that even when someone sets 
```
rbac_enabled: true
confluent_community: true
```
That rbac gets turned off. 

I am leaning toward putting this in the variables file and having an assertion, that complains thats not ok. WDYT?

Similar to the assertion in roles/confluent.control_center/tasks/main.yml